src/ds_caselaw_ingester/ingester.py

```diff
@@ -518,9 +498,9 @@ def will_publish(self) -> bool:
 
         # reparse
         if originator == "FCL":
-            if not self.existing_document_uri:
+            if not self.find_existing_document_by_ncn:
```

Comment:
I'm not 100% on this; why aren't we using `self.exists_in_database`?

Reply:
Tidied.